src/test/__fixtures/projects/complex-nesting-thingy/package1/test.jsx

```diff
@@ -0,0 +1,15 @@
+// @ts-nocheck
```

Comment:
I'm wondering if we can move some of this logic over to the "multiple instances of same dep" fixture instead. I feel like we have a tendency to use this complex fixture, but don't have a great handle on all of the edge cases it covers.

Reply:
https://github.com/ibm-telemetry/telemetry-js/issues/173